src/FSharp.Data.GraphQL.Server/Values.fs

```diff
@@ -312,6 +322,16 @@ let rec internal coerceVariableValue
             | Ok null when isNullable -> Ok null
             // TODO: Capture position in the JSON document
             | Ok null -> createNullError originalTypeDef
+            | Ok value when not isNullable ->
+                let ``type`` = value.GetType()
```

Comment:
it's probably a matter of taste, but I find the name
```
``type``
```
rather cumbersome. The backticks trick is a genius move for (some, mostly unit-test) methods, but as a variable name, as I said, it's IMHO cumbersome.
I guess `type_` should also do the trick.
Just my two cents; do as you wish 🙂

Reply:
In the other code bases I saw `ty` and `typ` but it looks bad as for me 🙂